src/Tools/ExternalAccess/Razor/EditorFeatures/Cohost/RazorCohostClientLanguageServerManagerFactory.cs

```diff
@@ -4,13 +4,14 @@
 
 using System;
 using System.Composition;
+using Microsoft.CodeAnalysis.ExternalAccess.Razor.Shared;
 using Microsoft.CodeAnalysis.Host.Mef;
 using Microsoft.CodeAnalysis.LanguageServer;
 
 namespace Microsoft.CodeAnalysis.ExternalAccess.Razor.Cohost;
 
 [Shared]
-[ExportCohostLspServiceFactory(typeof(IRazorCohostClientLanguageServerManager))]
+[ExportCohostLspServiceFactory(typeof(IRazorClientLanguageServerManager))]
```

Comment:
FYI, this makes this PR binary breaking so we need a dual insertion. Not necessarily the end of the world since this would be the 3rd in line, but wanted to note it at least. If this is the only break, it could be pretty trivially avoided if that was preferred.

Reply:
Ah yea. I can do without this change for now to avoid a binary break

Reply:
To be clear though, the new name makes sense, so perhaps export this twice, once with the old and once with the new name, so we can move over to it in time.